### ISSUE TITLE
Add callback_context handling

### ISSUE THREAD

**Issue by GibbsConsulting** [GibbsConsulting/django-plotly-dash#213]
Dash has a global `callback_context` variable available in a callback to provide access to information beyond the current callback argument values.

We can/should do at least some, and maybe all, of the following

- make this variable available in a dpd callback
- provide the information as extra arguments in an extended callaback
- make dpd extended values available in a similar manner

The dash approach does rely on this global variable not being impacted by threading etc, so providing an alternative to a global variable will be advantageous if asynchronous callbacks are added as per #158

**Comment by Dylan-Cunningham** [comment]
Would love to see this soon! My current workaround is to store a timestamp of each triggered button click. Then my callback displays the view of whichever was clicked most recently based on the stored timestamp. However, I believe this workaround is slowing my app significantly. Also, to get this functionality, I've had to add a callback for each button, and I had to add a hidden storage for each.

Anyone have a better workaround?
